<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Devendra" failures="2" tests="13" name="RetryMultipleTestCases" time="12.916" errors="0" timestamp="2022-12-29T14:06:55 IST">
  <testcase classname="TestNgHYR.TestClass" name="GoogleTest" time="7.845"/>
  <testcase classname="TestNgHYR.TestClass" name="method1" time="0.007">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="TestNgHYR.TestClass" name="method1" time="0.008">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="TestNgHYR.TestClass" name="method1" time="0.01">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="TestNgHYR.TestClass" name="method1" time="0.002">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="TestNgHYR.TestClass" name="method1" time="0.003">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="TestNgHYR.TestClass" name="method1" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNgHYR.TestClass.method1(TestClass.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- method1 -->
  <testcase classname="TestNgHYR.TestClass" name="method2" time="0.002">
    <skipped/>
  </testcase> <!-- method2 -->
  <testcase classname="TestNgHYR.TestClass" name="method2" time="0.002">
    <skipped/>
  </testcase> <!-- method2 -->
  <testcase classname="TestNgHYR.TestClass" name="method2" time="0.003">
    <skipped/>
  </testcase> <!-- method2 -->
  <testcase classname="TestNgHYR.TestClass" name="method2" time="0.001">
    <skipped/>
  </testcase> <!-- method2 -->
  <testcase classname="TestNgHYR.TestClass" name="method2" time="0.002">
    <skipped/>
  </testcase> <!-- method2 -->
  <testcase classname="TestNgHYR.TestClass" name="method2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNgHYR.TestClass.method2(TestClass.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- method2 -->
</testsuite> <!-- RetryMultipleTestCases -->
